.github/workflows/ci.yml

```diff
@@ -117,6 +117,19 @@ jobs:
         python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
         python builder.pyz build -p ${{ env.PACKAGE_NAME }} --cmake-extra=-DBUILD_SHARED_LIBS=ON
 
+  windows-app-verifier:
+    runs-on: windows-2022 # latest
+    steps:
+    - name: Build ${{ env.PACKAGE_NAME }} + consumers
+      run: |
+        python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
+        python builder.pyz build -p ${{ env.PACKAGE_NAME }} run_tests=false --cmake-extra=-DBUILD_TESTING=ON
+    - name: Run and check AppVerifier
+      run: |
+        echo "Starting to run AppVerifier on all tests found by CTest"
+        dir .\..\..\
+        python ..\..\aws-c-common\scripts\appverifier_ctest.py --build_directory .\aws-io-common\build\aws-io-common
```

Comment:
you can see where aws-c-common is from these logs: https://github.com/awslabs/aws-c-io/actions/runs/3061603397/jobs/4941581790#step:2:75
```
> git clone https://github.com/awslabs/aws-c-common.git D:\a\aws-c-io\aws-c-io\aws-c-io\build\deps\aws-c-common
Cloning into 'D:\a\aws-c-io\aws-c-io\aws-c-io\build\deps\aws-c-common'...
```

and earlier in the logs it prints:
```
Working in D:\a\aws-c-io\aws-c-io
```

so the path would be `.\aws-c-io\build\deps\aws-c-common\scripts\appverifier_ctest.py`